pages/dkp/kommander/2.3/appdeployments/index.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: layout.pug
+navigationTitle: AppDeployments
+title: AppDeployments
+excerpt: Use AppDeployments to deploy, and customize platform, DKP catalog, and custom applications in your environment
+menuWeight: 50
+beta: false
+enterprise: false
+---
+
+An `AppDeployment` is a [CustomResource][CRD] created by D2iQ with the purpose of deploying applications (platform, DKP catalog and custom applications) in management, managed clusters, or both. Customers of both Essential and Enterprise products use `AppDeployments`, regardless of their setup (networked, air-gapped, etc.), and their infrastructure provider.
```

Comment:
Since I did a pull-push, adding Matt's comment that got lost: 
```suggestion
An `AppDeployment` is a [CustomResource][CRD] created by DKP with the purpose of deploying applications (platform, DKP catalog and custom applications) in the management cluster, managed clusters, or both. Customers of both Essential and Enterprise products use `AppDeployments`, regardless of their setup (networked, air-gapped, etc.), and their infrastructure provider.
```